process/tsc_faq.md

```diff
@@ -17,31 +17,40 @@ The Technical Steering Committee (TSC) is the leadership of the project. This co
 
 ## Members
 
-Typically the TSC is comprised of the primary committers or maintainers in a project. To start there may be an appointed list, but over time more individuals may join the TSC according to the policies the TSC set for how new members are added. Regardless, TSC meetings are generally open to anyone interested unless there are sensitive issues to discuss which necessitate a private meeting.
+Typically, the TSC comprises the primary committers or maintainers in a project. To start, there may be an appointed list, but over time more individuals may join the TSC according to the policies the TSC set for how new members are added. Regardless, TSC meetings are generally open to anyone interested unless there are sensitive issues to discuss, which necessitate a private meeting.
 
-[LFX Project Control Center](https://projectadmin.lfx.linuxfoundation.org/) is a tool for managing project committees and roles within a project. Each project will be setup in that tool with a voting committee named 'Technical Steering Committee (TSC)', notating the project chairperson and all of the voting members. Project chairpersons will have access to manage this in addition to the Linux Foundation staff supporting the project. 
+[LFX Project Control Center](https://projectadmin.lfx.linuxfoundation.org/) is a tool for managing project committees and roles within a project. Each project will be set up in that tool with a voting committee named 'Technical Steering Committee (TSC),' notating the project chairperson and all voting members. Project chairpersons will have access to manage this in addition to the Linux Foundation staff supporting the project. Additionally, each project should have a [GitHub Team](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams) set up with each of their maintainers, with each project repository granted 'maintainer' permission. Generally, the LF Staff will have this setup for each project, but it's up to the project to maintain after that.
```

Comment:
Not sure that's generally known, especially the part about updating the LFX platform. Would be good to review all current projects and make sure that's up to date (unless that's already been happening).

Reply:
Yep - we are doing that in the background. Once we transition over then we will make sure TSC leaders are all enabled.